package/src/components/MessageList/__tests__/__snapshots__/MessageList.test.js.snap

```diff
@@ -931,7 +931,7 @@ exports[`MessageList should render the message list and match snapshot 1`] = `
               ]
             }
           >
-            Sep 3
+            Sep 7
```

Comment:
These snapshots are using dynamic date... We should fix that ASAP in order to avoid the need to update them @vishalnarkhede